specs/_features/maxeb_increase/capella.py

```diff
@@ -842,7 +845,7 @@ def is_slashable_validator(validator: Validator, epoch: Epoch) -> bool:
     """
     Check if ``validator`` is slashable.
     """
-    return (not validator.slashed) and (validator.activation_epoch <= epoch < validator.withdrawable_epoch)
+    return (not is_slashed_attester(validator)) and (validator.activation_epoch <= epoch < validator.withdrawable_epoch)
```

Comment:
I think we should replace this function with the two: `is_slashable_attester` and `is_slashable_proposer` to make them independent